test/functional/rpc_net.py

```diff
@@ -163,11 +163,11 @@ def _test_getpeerinfo(self):
 
         # Check dynamically generated networks list in getpeerinfo help output.
         assert "(ipv4, ipv6, onion, i2p, not_publicly_routable)" in self.nodes[0].help("getpeerinfo")
-        assert_equal(peer_info[0][0]['connection_type'], 'inbound')
-        assert_equal(peer_info[0][1]['connection_type'], 'manual')
+        assert_equal(peer_info[0][0]['connection_type'], 'manual')
```

Comment:
ok

Reply:
```suggestion
        # This part is slightly different comparing to the Bitcoin implementation. This is expected because we create connections on network setup a bit differently too.
        # We also create more connection during the test itself to test mn specific stats
        assert_equal(peer_info[0][0]['connection_type'], 'manual')
```